FirebaseAnalyticsOnDeviceConversion.podspec

```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
 
     s.cocoapods_version = '>= 1.10.2'
 
-    s.dependency 'GoogleAppMeasurementOnDeviceConversion', '9.0'
+    s.dependency 'GoogleAppMeasurementOnDeviceConversion', '~> 9.0'
```

Comment:
Question, @paulb777. Should I add the `~>` operator? 

I was thinking yes because, for example, FirebaseAuth 9.1.0 has a dependency on FirebaseCore ~> 9.0.

Reply:
Actually FirebaseAnalytics requires matching GoogleAppMeasurement versions, so we should update the dependency version with every release, just as we do the GoogleAppMeasurement versions in the FirebaseAnalytics.podspec.json.

cc: @htcgh @tsunghung

Reply:
Noted and fixed (55b44be). Unfortunately, this will require a workaround in our tooling. We couldn't keep `[FirebaseAnalyticsOnDeviceConversion.podspec](https://github.com/firebase/firebase-ios-sdk/pull/9787/files/8f54ea297b881ac3030143c391c63d60bcb1e0cf#diff-08afef50b0f0fc9cfd1744148a886f614950cdad0e3afae3fb91c9441ce3c2e5)` as a `json` spec, because of its `source` attribute. Now, as a Ruby spec, it's `s.dependency 'GoogleAppMeasurementOnDeviceConversion', '9.0'` will need to be bumped every release.

Can the Analytics** and GoogleAppMeasurement** podspecs be converted to Ruby files? That would simplify release tooling since both `json` and `podspec` formats would need to be maintained?

Reply:
I'd be ok with making all the podspecs ruby.

Reply:
Ok, sounds good.